tests/worker_deployment_test.go

```diff
@@ -264,6 +264,15 @@ func (s *WorkerDeploymentSuite) TestDescribeWorkerDeployment_TwoVersions() {
 	secondVersion := tv.WithBuildIDNumber(2)
 
 	go s.pollFromDeployment(ctx, firstVersion)
+
+	// waiting for 3 seconds to start the second version later.
+	startTime := time.Now()
+	waitTime := 3 * time.Second
```

Comment:
can you make waitTime 1 ms instead? that will be enough that we can still tell sorting, without adding too much time to the total test time